smoke-test/tests/utils.py

```diff
@@ -169,14 +169,7 @@ def delete(entry):
         d = json.load(f)
         Parallel(n_jobs=10)(delayed(delete)(entry) for entry in d)
 
-    # Deletes require 60 seconds when run between tests operating on common data, otherwise standard sync wait
-    if shared_data:
-        wait_for_writes_to_sync()
-#        sleep(60)
-    else:
-        wait_for_writes_to_sync()
-#        sleep(requests.ELASTICSEARCH_REFRESH_INTERVAL_SECONDS)
-
+    wait_for_writes_to_sync()
```

Comment:
This is insufficient, after consumption there needs and additional elasticsearch refresh interval seconds. This is configurable and currently defaulting to 1 second. For stability, I would modify the function to add this to the kafka check.